recipes/pygnome/meta.yaml

```
@@ -0,0 +1,103 @@
+{% set name = "PyGnome" %}
+{% set version = "1.1.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NOAA-ORR-ERD/PyGnome/archive/v{{ version }}.tar.gz
+  sha256: ad14d089932098bcd0b063f4c08fb6a0836de4747edec0497397671f1830e03b
+
+build:
+  skip: True  # [py<39 or py>311 or python_impl == 'pypy']
+  missing_dso_whitelist:
+    - "*/lib_gnome.dll"         # [win]
+    - "*/point_in_polygon.dll"  # [win]
+  number: 0
+  script: cd py_gnome && {{ PYTHON }} -m pip install ./ -vv
+
+requirements:
+  build:
+    - python                              # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - numpy
+    - cython
+    - netCDF4
+    - scikit-build-core
+    - clang 15.*             # [osx]
+    - {{ compiler('c') }}    # [not osx]
+    - {{ stdlib('c') }}      # [not osx]
+    - {{ compiler('cxx') }}  # [not osx]
+    - make
+    - cmake
+  host:
+    - python
+    - pip
+    - numpy
+    - scikit-build-core
+    - blosc
+    - bzip2
+    - curl
+    - hdf4
+    - hdf5
+    - libcurl
+    - libnetcdf
+    - libaec
+    - libxml2
+    - libzip
+    - openssl
+    - zlib
+    - zstd
+  run:
+    # Even though we have python defined in reqs->host, the windows build is
+    # still complaining about it not being in reqs->run.
+    - python  # [win]
```

Python packages should always have an explicit `python` in  `run` 
```suggestion
    - python
```